lib/query/query_supporters.rb

```diff
@@ -530,6 +530,21 @@ def self.dupes_on_name_and_email(np_id)
       .map(&:flatten)
   end
 
+  def self.dupes_on_name_and_phone(np_id)
+    dupes_expr(np_id)
+      .and_where(
+        "name IS NOT NULL\
```

Comment:
Should we also be checking if `name` is blank?

Reply:
Makes sense!

Reply:
I noticed we don't do that in other methods, I wonder if there's a reason.

Reply:
I think we should have `name != ''` here as the code currently says but this does worry me if we don't do it in any of the other methods.

I would have been nice to know why this decision was made in the past. I guess make sure you're doing a bit of testing of this to make sure things are fine.

Reply:
I smoke tested and added a few tests to make sure everything was fine.